summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/ScaldingPlatform.scala

```diff
@@ -652,67 +652,45 @@ class Scalding(
   def plan[T](prod: TailProducer[Scalding, T]): PipeFactory[T] =
     Scalding.plan(options, prod)
 
-  protected def ioSerializations: List[Class[_ <: HSerialization[_]]] = List(
-    classOf[org.apache.hadoop.io.serializer.WritableSerialization],
-    classOf[cascading.tuple.hadoop.TupleSerialization],
-    classOf[com.twitter.chill.hadoop.KryoSerialization]
-  )
-
   def withRegistrars(newRegs: List[IKryoRegistrar]) =
     new Scalding(jobName, options, transformConfig, newRegs ++ passedRegistrars)
 
-  def withConfigUpdater(fn: SummingbirdConfig => SummingbirdConfig) =
+  def withConfigUpdater(fn: Config => Config) =
     new Scalding(jobName, options, transformConfig.andThen(fn), passedRegistrars)
 
-  def updateConfig(conf: Configuration) {
-    val scaldingConfig = SBChillRegistrar(ScaldingConfig(conf), passedRegistrars)
-    Scalding.logger.debug("Serialization config changes:")
-    Scalding.logger.debug("Removes: {}", scaldingConfig.removes)
-    Scalding.logger.debug("Updates: {}", scaldingConfig.updates)
+  def configProvider(hConf: Configuration): Config = {
+    import com.twitter.scalding._
+    import com.twitter.chill.config.ScalaMapConfig
+    val conf = Config.hadoopWithDefaults(hConf)
 
-    // now let the user do her thing:
-    val transformedConfig = transformConfig(scaldingConfig)
+    val kryoReg = new IterableRegistrar(passedRegistrars)
 
-    Scalding.logger.debug("User+Serialization config changes:")
-    Scalding.logger.debug("Removes: {}", transformedConfig.removes)
-    Scalding.logger.debug("Updates: {}", transformedConfig.updates)
+    val initKryo = conf.getKryo match {
+      case None =>
+        new serialization.KryoHadoop(ScalaMapConfig(conf.toMap))
+      case Some(kryo) => kryo
+    }
+    conf
+      .setSerialization(
+        Left((classOf[serialization.KryoHadoop], initKryo.withRegistrar(kryoReg))), Nil)
+  }
+
+  final def buildConfig(hConf: Configuration): Config = {
+    val config = transformConfig(configProvider(hConf))
 
-    transformedConfig.removes.foreach(conf.set(_, null))
-    transformedConfig.updates.foreach(kv => conf.set(kv._1, kv._2.toString))
     // Store the options used:
-    conf.set("summingbird.options", options.toString)
-    conf.set("summingbird.jobname", jobName)
-    // legacy name to match scalding
-    conf.set("scalding.flow.submitted.timestamp",
-      System.currentTimeMillis.toString)
-    conf.set("summingbird.submitted.timestamp",
-      System.currentTimeMillis.toString)
-
-    def ifUnset(k: String, v: String) { if (null == conf.get(k)) { conf.set(k, v) } }
-    // Set the mapside cache size, this is important to not be too small
-    ifUnset("cascading.aggregateby.threshold", "100000")
+    val postConfig = config.+("summingbird.options" -> options.toString)
+      .+("summingbird.jobname" -> jobName)
+      // legacy name to match scalding
```

Comment:
I don't think this is needed anymore (I think scalding already puts in the scalding.flow.submitted.timestamp).